src/components/organisms/ValidationPane/OpenPolicyAgent/useOpenPolicyAgentTable.tsx

```diff
@@ -63,21 +63,21 @@ export function useOpenPolicyAgentTable() {
       },
       {
         key: 'severity',
-        title: 'Severity',
+        title: `${width < 420 ? '' : 'Severity'}`,
```

Comment:
Can we set 420 in constant somewhere so it's easier to change and can we have a descriptive name for it?